Components/Inputs.css

```diff
@@ -0,0 +1,13 @@
+.input-email,.input-pwd{
```

Comment:
add a space after coma. Increases readability.

Reply:
sure will do it.